qiskit_metal/renderers/renderer_ansys/ansys_renderer.py

```diff
@@ -1029,6 +1043,63 @@ def add_mesh(self):
                 self.assign_mesh,
                 MaxLength=self._options['max_mesh_length_jj'])
 
+    #Still implementing
+    def auto_wirebonds(self, table):
+        """
+        Adds wirebonds to the Ansys model for path elements where;
+        subtract = True and wire_bonds = True.
+        Uses render options for determining of the;
+        * wb_threshold -- the minimum distance between two vertices of a path for a
+            wirebond to be added.
+        * wb_offset -- offset distance for wirebond placement (along the direction
```

Comment:
If wb_offset is constant for all wire bonds, what is it used for? Wouldn't we want this offset to apply to different wire bonds relative to each other?

Reply:
It is for a simple manner to stagger the wirebonds, in case have tight meandering and the wirebonds collide if left at symmetry points.